js/__internal/grids/grid_core/filter_panel/module.ts

```diff
@@ -41,14 +43,18 @@ const FilterPanelView = modules.View.inherit({
       .addClass(that.addWidgetPrefix(FILTER_PANEL_LEFT_CONTAINER))
```

Comment:
Question.

Should we create a filter panel if ```isColumnDefined``` is false?

Reply:
Let's discuss it on the squad meeting